corehq/toggles.py

```diff
@@ -1731,18 +1731,15 @@ def _commtrackify(domain_name, toggle_is_enabled):
     relevant_environments={'icds', 'india'},
 )
 
-RATE_LIMIT_SUBMISSIONS = DynamicallyPredictablyRandomToggle(
-    'rate_limit_submissions',
-    'Rate limit submissions with a 429 TOO MANY REQUESTS response',
+DO_NOT_RATE_LIMIT_SUBMISSIONS = StaticToggle(
+    'do_not_rate_limit_submissions',
+    'Do not rate limit submissions for this project, on a temporary basis.',
     TAG_INTERNAL,
     [NAMESPACE_DOMAIN],
     description="""
-    While we are gaining an understanding of the effects of rate limiting,
-    we want to force rate limiting on certain domains, while also being to
-    toggle on and off global rate limiting quickly in response to issues.
-
-    To turn on global rate limiting, set Randomness Level to 1.
-    To turn it off, set to 0.
+    When an individual project is having problems with rate limiting,
+    use this toggle to lift the restriction for them on a temporary basis,
+    just to unblock them while we sort out the conversation with the client.
```

Comment:
What will keep people from setting this flag and then forgetting to unset it later? It seems like it might be nice if there were a way to disable rate limiting for a week, for example, and have it automatically re-enabled after that period. I suppose that would be more work since we don't have a concept of auto-reset feature flags...

Reply:
Yeah that's a fair question, and you pretty much hit the answer. This is easier, and until it becomes a problem, I'm going to trust people and just check this periodically. It's possible we'll settle on a longer term solution before it becomes a problem, so I wanted to give an escape hatch here without designing something even remotely elaborate.